### PR TITLE
Use main in import strings workflow

### PR DESCRIPTION
**Pull request by flodolo**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [submodule "i18n"]
 	path = i18n
 	url = https://github.com/mozilla-l10n/mozilla-vpn-client-l10n.git
-	branch= master
+	branch= main
```
Comment (flodolo) [inline .gitmodules:4]: Technically, I think we could remove this if we just want the default branch?

**Comment by flodolo** [comment]
Note: unfortunately I can't really test automation from a fork :-(

File: .gitmodules [hunk 2]
```diff
@@ -1,7 +1,6 @@
 [submodule "i18n"]
 	path = i18n
 	url = https://github.com/mozilla-l10n/mozilla-vpn-client-l10n.git
-	branch= master
```
Comment (bakulf) [inline .gitmodules:4]: I would add `branch = main` because, by default, git uses "master".

**Review by bakulf — APPROVED** [review]

**Comment by flodolo** [comment]
Should we try merging and check if automation works as expected? Alternatively, I can push to a branch in this repo and verify it there.

**Comment by flodolo** [comment]
Seems to work https://github.com/mozilla-mobile/mozilla-vpn-client/pull/742

**Pull request merged**